src/pages/test-results-search/components/advanced-search/advanced-search.ts

```diff
@@ -46,7 +46,8 @@ export class AdvancedSearchComponent {
     const advancedSearchParams: AdvancedSearchParams = {
       startDate: this.startDate,
       endDate: this.endDate,
-      staffNumber: this.staffNumber,
+      // We need to remove any leading zeros from the staff number
```

Comment:
Create a helper function with the name of `removingLeadingZeros` and call that instead of this comment